db/seeds.rb

```diff
@@ -5,3 +5,81 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+puts 'Seeding users'
```

Comment:
As mentioned in the commit message for this commit, most projects I have worked on we seed some data to manually review the UI and at times we have used them for the occasional testing fixture.

Normally I would like to use the [factorybot gem](https://github.com/thoughtbot/factory_bot) especially to seed records with associations.